lightning/src/routing/router.rs

```diff
@@ -171,7 +172,7 @@ impl< G: Deref<Target = NetworkGraph<L>> + Clone, L: Deref, ES: Deref, S: Deref,
 	fn create_blinded_paths<
 		T: secp256k1::Signing + secp256k1::Verification
 	> (
-		&self, recipient: PublicKey, peers: Vec<PublicKey>, secp_ctx: &Secp256k1<T>,
+		&self, recipient: PublicKey, peers: Vec<message::ForwardNode>, secp_ctx: &Secp256k1<T>,
```

Comment:
If we're going to split this into two methods, do we still need `message::ForwardNode`? Can't one method take pubkeys and the other take scids?

Reply:
We alwasys need the pubkeys when creating the blinded path, so we'd need to either make a new struct or use a tuple. It's kinda nice having it use an `Option` for `short_channel_id`, though. It makes it easy in `ChannelManager::create_blinded_path` to fallback to `None` if a usable channel can't be found.